packages/dds/merge-tree/src/test/revertibles.spec.ts

```diff
@@ -235,4 +239,107 @@ describe("MergeTree.Revertibles", () => {
 
         logger.validate();
     });
+
+    describe("Redo does not works as expected when a pair of markers and text is involved", ()=>{
```

Comment:
seems negated from what it should be...?

```suggestion
    describe("Redo works as expected when a pair of markers and text is involved", ()=>{
```